src/components/TransactionModal/index.tsx

```diff
@@ -101,8 +103,10 @@ export function UpdateTransactionModal({
         <button type="submit">
           {formikValue.isSubmitting ? (
             <CircularProgress color="inherit" />
-          ) : (
+          ) : node ? (
             'Atualizar'
+          ) : (
+            'Criar'
```

Comment:
Avoid nested ternaries, it's better to have a simple function:

```ts

const renderSubmitContent = () => {
  if (formikValue.isSubmitting) {
    return <CircularProgress color="inherit" />
  }

  if (node) {
    return 'Atualizar';
  }

  return 'Criar';
}
```

Reply:
Thanks, that's better to read. Going to fix it.